Source/WebCore/Modules/WebGPU/GPUComputePassEncoder.cpp

```diff
@@ -76,6 +76,10 @@ void GPUComputePassEncoder::setBindGroup(GPUIndex32 index, const GPUBindGroup& b
     GPUSize64 dynamicOffsetsDataStart,
     GPUSize32 dynamicOffsetsDataLength)
 {
+    auto offset = checkedSum<uint64_t>(dynamicOffsetsDataStart, dynamicOffsetsDataLength);
```

Comment:
Is this all the validation that will need to happen here? I wonder if it needs to be shared instead of copied between the three encoders, but if this is all, then maybe it doesn't matter.